README.md

```diff
@@ -36,25 +36,25 @@ _✨ A modern Anki [custom scheduling](https://faqs.ankiweb.net/the-2021-schedul
 
 # Introduction
 
-FSRS4Anki consists of two main parts: scheduler and optimizer.
+FSRS4Anki consists of two main parts: the scheduler and the optimizer.
 
-- The scheduler replaces the Anki's built-in scheduler and schedules the cards according to the FSRS algorithm.
-- The optimizer uses machine learning to learn your memory patterns and finds parameters that provide the best fit to your review history. For details about the working of the optimizer, please read [the mechanism of optimization](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-mechanism-of-optimization).
+- The scheduler replaces Anki's built-in scheduler and schedules the cards according to the FSRS algorithm.
+- The optimizer uses machine learning to learn your memory patterns and finds parameters that best fit your review history. For details about the working of the optimizer, please read [the mechanism of optimization](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-mechanism-of-optimization).
 
 For details about the FSRS algorithm, please read [the algorithm](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm). If you are interested, you can also read my papers:
 - [A Stochastic Shortest Path Algorithm for Optimizing Spaced Repetition Scheduling](https://www.maimemo.com/paper/) (free access), and
-- [Optimizing Spaced Repetition Schedule by Capturing the Dynamics of Memory](https://www.researchgate.net/publication/369045947_Optimizing_Spaced_Repetition_Schedule_by_Capturing_the_Dynamics_of_Memory) (submit request).
+- [Optimizing Spaced Repetition Schedule by Capturing the Dynamics of Memory](https://www.researchgate.net/publication/369045947_Optimizing_Spaced_Repetition_Schedule_by_Capturing_the_Dynamics_of_Memory) (submit a request).
 
 FSRS4Anki Helper is an Anki add-on that complements the FSRS4Anki Scheduler. You can read about it here: https://github.com/open-spaced-repetition/fsrs4anki-helper
 
 # How to Get Started?
 
 中文版请见：[FSRS4Anki 使用指北](https://zhuanlan.zhihu.com/p/636564830)
 
-To get started with FSRS, you'll need to follow a two-step process.
+To get started with FSRS, you must follow a two-step process.
 
-- First, you'll need to enable the FSRS scheduler in your Anki application.
-- Next, you'll need to personalize FSRS to suit your learning patterns.
+- First, you must enable the FSRS scheduler in your Anki application.
+- Next, you must personalize FSRS to suit your learning patterns.
```

Comment:
The "must" feels forceful here. And new users might not have enough data to personalize FSRS.

Reply:
> The "must" feels forceful here. And new users might not have enough data to personalize FSRS.

Ok. Reverted this change.